### ISSUE TITLE
A question. I actulay have no idea.

### ISSUE THREAD

**Issue by Nani-Sore-0** [gordonmzhu/questions#7]
### What is the associated Watch and Code lesson? 

https://watchandcode.com/courses/60264/lectures/1181252 This is the video that the problem started to appear. I have no idea how.

### Include a live demo link so that I can run your code. 

https://plnkr.co/edit/an6gr1J6GpfRlrZ8Ki0A?p=preview

### Explain the exact steps I need to take to replicate the problem. 

To replicate add a few to todos and delete the first one and you will see all todos get removed. If i add lets say 5 todos and remove the second one, form the second one down all get removed. So all except the first. It is like that when a todo is removed it removes all todos under it.

### What did you expect to happen? 

When I click the button that todo gets removed and the DOM updates.

### What actually happened? 

No error. What actually happened see above.

### Describe what you've done so far to troubleshoot the problem. 

I have rewritten those parts of the code a few times I ran the debugger a lot a lot like for so long I think this might be a blessing in disguise because I understand the code quite well now. I have actually no idea how this is happening it dose it job and then it loops for some reason I just can't understand. Thanks m8 quality course right here tho.

**Comment by gordonmzhu** [comment]
You've skipped a very important step from version 10. Go back to the beginning of version and carefully redo the lessons and you will catch what you missed. At the end of every video, triple-check to make sure you have the same code as mine (you definitely do not at the moment).

**Comment by Nani-Sore-0** [comment]
Thanks for the reply.
I watched it all again twice but it is all the same except the last line
and i suspect that might be the problem but that leads to my original
question I can't put the " view.setUpEventListener(); " in the last line
because the ul element is empty and it gives my a type error: todosUl is
null. I really don't know anymore just tell me please. Thanks.

**Comment by gordonmzhu** [comment]
Why can't you put `view.setUpEventListener()`? I don't quite understand what you mean. Can you show a demo that shows this behavior?

**Comment by Nani-Sore-0** [comment]
Wow thanks for taking the so much time to respond to me. 

https://plnkr.co/edit/an6gr1J6GpfRlrZ8Ki0A?p=preview (last line add)
https://plnkr.co/edit/an6gr1J6GpfRlrZ8Ki0A?p=preview (old plunk)

All that I have changed is the only thing I see different from my code to yours.
I put the " view.setUpEventListener(); " in the last line instead of the " handlers.addTodo ". I have used this a workaround for the problem of getting a error when I try to set up a event listener on an empty ul list.
